torchx/specs/named_resources_aws.py

```diff
@@ -348,6 +348,46 @@ def aws_trn1_32xlarge() -> Resource:
     )
 
 
+def aws_inf2_xlarge() -> Resource:
+    return Resource(
+        cpu=4,
+        gpu=0,
+        memMB=32 * GiB,
```

Comment:
16GB of memory not 32? Pretty sure this refers to the CPU memory not the accelerator memory

Reply:
@ashvinnihalani  Good catch, thanks for pointing this out.

I was referencing trn1 and thought the field is for GPU memory, but then realize trn1 has same GPU and CPU memory.

Reply:
@ashvinnihalani Updated